modules/ipi-install-bmc-addressing-for-dell-idrac.adoc

```diff
@@ -6,7 +6,7 @@
 [id='bmc-addressing-for-dell-idrac_{context}']
 = BMC addressing for Dell iDRAC
 
-The `address` field for each `bmc` entry is a URL for connecting to the {product-title} cluster nodes, including the type of controller in the URL scheme and its location on the network.
+The `address` configuration setting for each `bmc` entry is a URL for connecting to the {product-title} cluster nodes, including the type of controller in the URL scheme and its location on the network. The `username` configuration for each `bmc` entry must specify a user with `Administrator` privileges.
```

Comment:
Does `Administrator` priveleges need mark up or the capital A? I don't see any other instances of `Administrator` priveleges.

Reply:
In the Dell documentation they mention as `Administrator` so we should try to follow that. 

https://www.dell.com/support/manuals/pt-br/idrac9-lifecycle-controller-v3.3-series/idrac_3.30.30.30_ug/idrac-user-roles-and-privileges?guid=guid-d2ed6a79-c1b0-4342-93d1-22c731f6e087&lang=en-us 

The change looks good thanks @johnwilkins !

Reply:
It is Administrator. I checked on my Dell servers, and it's the setting.